vignettes/visits_periods.Rmd

```diff
@@ -247,26 +374,22 @@ adsl
 
 ## Creating BDS and OCCDS Treatment Variables {#treatment_bds}
 
-If a period reference dataset is available, BDS and OCCDS variables for
+If a period/phase reference dataset is available, BDS and OCCDS variables for
 treatment can be created by calling `derive_vars_joined()`.
 
 For example the variables `APERIOD` and `TRTA` can be derived from the
 period reference dataset defined above.
 
 ```{r}
 adae <- tribble(
-  ~USUBJID, ~ASTDT,
-  "1",      "2021-01-05",
-  "1",      "2021-02-05",
-  "1",      "2021-03-05",
-  "1",      "2021-04-05",
-  "2",      "2021-02-15",
-  "2",      "2021-03-10",
-) %>%
-  mutate(
-    ASTDT = ymd(ASTDT),
```

Comment:
I think we should keep this line. It seems to work but it looks strange if `ASTDT` is character instead of date.